x-pack/plugins/actions/server/routes/connector/get_all/get_all.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IRouter } from '@kbn/core/server';
+import { transformGetAllConnectorsResponse } from './transforms';
+import { ActionsRequestHandlerContext } from '../../../types';
+import { BASE_ACTION_API_PATH } from '../../../../common';
+import { ILicenseState } from '../../../lib';
+import { verifyAccessAndContext } from '../../verify_access_and_context';
+
+export const getAllConnectorsRoute = (
+  router: IRouter<ActionsRequestHandlerContext>,
+  licenseState: ILicenseState
+) => {
+  router.get(
+    {
+      path: `${BASE_ACTION_API_PATH}/connectors`,
+      validate: {},
+    },
+    router.handleLegacyErrors(
+      verifyAccessAndContext(licenseState, async function (context, req, res) {
+        const actionsClient = (await context.actions).getActionsClient();
+        const result = await actionsClient.getAll();
+        return res.ok({
+          body: transformGetAllConnectorsResponse(result),
```

Comment:
Let's use the V1 version of the transform, it will let us know when the response schema has changed.

Also I like to version the body explicitly:

```
const responseBody: ConnectorResponseV1 = transformGetAllConnectorsResponse(result)
```

Reply:
Thanks for reviewing Jiawei. I addressed all the comments.